tests/unit/test_views.py

```diff
@@ -29,8 +29,12 @@ def sendVariantSetsSearch(self, data):
                              headers={'Content-type': 'application/json'},
                              data=data)
 
-    def testServer(self):
-        self.assertEqual(404, self.app.get('/doesNotExist').status_code)
+    def test404sReturnJson(self):
+        response = self.app.get('/doesNotExist')
+        exception = protocol.GAException.fromJsonString(
+            response.get_data())
+        self.assertEqual(404, response.status_code)
+        self.assertEqual(5, exception.errorCode)
```

Comment:
Can you add a TODO here to indicate we need to get rid of the hard coded GA error codes? HTTP error codes are fine as these are familiar.